xml/System/MathF.xml

```diff
@@ -463,8 +463,7 @@ The static fields and methods of the `MathF` class correspond to those of the <x
  -or-  
   
  <see cref="F:System.Single.NaN" /> if <paramref name="x" /> equals <see cref="F:System.Single.NaN" />.</returns>
-        <remarks>
-        </remarks>
+        <remarks></remarks>
```

Comment:
This seems like something the tool can fix - if it finds an empty `<remarks>` element, it should be filled with the `To be added.` placeholder:

```suggestion
        <remarks>To be added.</remarks>
```

Reply:
Do we want `To be added` for empty remarks, what if there are none to add?

Reply:
The `To be added.` is the default placeholder and should stay there.

Reply:
Even though it wasn't there before this change? Several of these are existing entries that the tool simply changed from
```xml
<remarks>
</remarks>
```
to
```xml
<remarks></remarks>
```